public/javascripts/Admin/build/Admin.js

```diff
@@ -1,6 +1,211 @@
-function Admin (_, $, c3, turf) {
```

Comment:
I don't think you made changes to Admin, right? I cannot figure out if these changes are significant. If not, please get rid of this from the PR.